website/docs/components/form/super-select/partials/specifications/states.md

```diff
@@ -0,0 +1,23 @@
+## States
+
+!!! Info
+
+**A note on disabled states**
+
+Because disabled states completely remove the interactive function of an element, it can be challenging for a user to understand why it has been disabled and/or why they cannot interact with that element. In an effort to avoid this confusion, we opt for using methods like displaying or hiding the element and, thus, are not offering disabled states for the Dropdown. Read more about [when to enable vs hide](https://hashicorp.atlassian.net/wiki/spaces/DES/pages/2678685874/Hiding+Disabling).
+!!!
+
+### Trigger
+
+![Showing the different states for the SuperSelect trigger](/assets/components/form/super-select/super-select-trigger-states.png =800x*)
+
+### ListItem
+
+#### Checkmark
+
+![Showing the different states for the SuperSelect list item checkmark variant](/assets/components/form/super-select/super-select-listitem-checkmark-states.png =432x*)
```

Comment:
Looking at the CSS, I _think_ the disabled version of the checkmark doesn't look like this. @KristinLBradley / @alex-ju can you double check?

Reply:
I found a reference in the docs on setting disabled options: https://ember-power-select.com/docs/the-list (Originally I was looking in the API docs and not finding anything) I will add an example to the Showcase.

Reply:
Added to the Showcase in the component branch

Reply:
When an option is disabled it can't be manually selected. However, it seems that it can be preselected using `@selected`.

We don't have a special visual style for this. Should it perhaps be both "checked" and use a grey text color (vs. blue)?

Reply:
@alex-ju pushed up a number of updates related to this. I also pushed up some fixes to styles that hadn't been working properly.